components/formats-bsd/test/loci/formats/utests/SixteenBitLosslessJPEG2000Test.java

```diff
@@ -66,12 +66,21 @@
   private static final Logger LOGGER =
     LoggerFactory.getLogger(SixteenBitLosslessJPEG2000Test.class);
 
+  private int increment = 1024;
+
+  public SixteenBitLosslessJPEG2000Test() {
+  }
```

Comment:
a more idiomatic pattern might be to have `private final int increment;` and in this constructor do `this(1024);` but this approach is fine too